src/main/scala/no/ndla/learningpathapi/controller/LearningpathControllerV2.scala

```diff
@@ -656,18 +657,33 @@ trait LearningpathControllerV2 {
       }
     }
 
-    val getContributors =
+    private val getContributors =
       (apiOperation[List[Author]]("getContributors")
         summary "Fetch all previously used contributors in learningpaths"
         notes "Retrieves a list of all previously used contributors in learningpaths"
-        parameters (asHeaderParam[Option[String]](correlationId))
+        parameters asHeaderParam[Option[String]](correlationId)
         responseMessages response500
         authorizations "oauth2")
 
     get("/contributors/", operation(getContributors)) {
       readService.contributors
     }
 
-  }
+    private val withStatus: SwaggerSupportSyntax.OperationBuilder =
+      (apiOperation[List[LearningPathV2]]("withStatus")
+        summary "Fetch all learningpaths with specified status"
+        notes "Fetch all learningpaths with specified status"
+        parameters (asHeaderParam[Option[String]](correlationId),
+        asQueryParam[String](learningPathStatus))
+        responseMessages (response500, response400)
+        authorizations "oauth2")
+    get("/with-status/", operation(withStatus)) {
```

Comment:
Foreslår å kalle dette endepunktet for `/status/:STATUS` og gjøre om `status` til et path parameter siden den alltid skal være definert. Kan også vurdere å flytte denne så den ligger under eller over `/:learningpath_id/status/`